lib/skate_web/channels/vehicles_search_channel.ex

```diff
@@ -0,0 +1,76 @@
+defmodule SkateWeb.VehiclesSearchChannel do
```

Comment:
Some duplication here of the existing `VehiclesChannel` search topic - key new piece is `handle_in`